financial-connections/src/main/java/com/stripe/android/financialconnections/features/linkaccountpicker/LinkAccountPickerViewModel.kt

```diff
@@ -111,7 +104,7 @@ internal class LinkAccountPickerViewModel @AssistedInject constructor(
                 defaultCta = display.defaultCta,
                 consumerSessionClientSecret = consumerSession.clientSecret,
                 singleAccount = manifest.singleAccount,
-                selectedAccountIds = selectedAccountIds
+                selectedAccountIds = emptyList()
```

Comment:
Do we really want to disable it for _all_ accounts, even valid ones? I feel like we should only avoid selecting accounts that show a drawer on selection.

Reply:
That was option B! I like it better (to make it perfect I'll do a follow-up backend PR resorting accounts so that those with drawer are below the real selectable ones) - https://github.com/stripe/stripe-android/pull/8945/commits/9d9a4526c425b576e0bc67cc622d57063a30b558